lib/cf_bot.js

```diff
@@ -98,7 +98,9 @@ class CFBot {
     this.events_monitor.start()
 
     // refresh tokens ten minutes before they run out.
-    var timeout = (tokens.expires_in - (60 * 10)) * 1000
```

Comment:
I'd prefer to modify this line to 
```
var timeout = (tokens.expires_in - 60) * 1000
```

to turn the timeout down to a single minute rather than the code below.